files/image_config/hostcfgd/hostcfgd

```diff
@@ -263,48 +311,23 @@ class HostConfigDaemon:
         self.iptables.iptables_handler(key, data, add)
 
     def feature_status_handler(self, key, data):
-        status_data = self.config_db.get_table('FEATURE')
-        for key in status_data.keys():
-            if not key:
-                syslog.syslog(syslog.LOG_WARNING, "FEATURE key is missing")
-                continue
+        feature_name = key
+        feature_table = self.config_db.get_table('FEATURE')
+        if feature_name not in feature_table.keys():
```

Comment:
Just curious, is it possible to delete a feature from `feature_table` after it is enabled?

Reply:
No. The concept is that "features" are built into the image. They cannot be removed, only disabled.